components/org.wso2.bfsi.consent.management.common/src/main/java/org/wso2/bfsi/consent/management/common/config/ConsentManagementConfigParser.java

```diff
@@ -196,6 +198,16 @@ private String replaceSystemProperty(String text) {
         return textBuilder.toString();
     }
 
+    /**
+     * Method to obtain map of configs.
+     *
+     * @return Config map
+     */
+    public Map<String, Object> getConfiguration() {
+
+        return configuration;
+    }
```

Comment:
wdyt?

```suggestion
    public Map<String, Object> getConfigurations() {

        return this.configurations;
    }
```

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/5/commits/4d9138c332549a29ab817615c49fd31c6d8cc121